src/Runner.Worker/ExecutionContext.cs

```diff
@@ -872,7 +872,7 @@ public void QueueSummaryFile(string name, string filePath, Guid stepRecordId)
                 throw new FileNotFoundException($"Can't upload (name:{name}) file: {filePath}. File does not exist.");
             }
 
-            _jobServerQueue.QueueSummaryUpload(stepRecordId, name, filePath, deleteSource: false);
+            _jobServerQueue.QueueResultsUpload(stepRecordId, name, filePath, ChecksAttachmentType.StepSummary, deleteSource: false, finalize: false, firstBlock: false);
```

Comment:
the `finalize: false, firstBlock: false` feels odd, since you are done with uploading the only block for summary.
should it be `finalize: true, firstBlock: true`?